include/crm/common/action_relation_internal.h

```diff
@@ -148,4 +148,32 @@ typedef struct {
 
 typedef struct pe_action_wrapper_s pcmk__related_action_t;
 
+/*!
+ * \internal
+ * \brief Set action relation flags
+ *
+ * \param[in,out] ar_flags      Flag group to modify
+ * \param[in]     flags_to_set  enum pcmk__action_relation_flags to set
+ */
+#define pcmk__set_relation_flags(ar_flags, flags_to_set) do {           \
+        ar_flags = pcmk__set_flags_as(__func__, __LINE__, LOG_TRACE,    \
+                                      "Action relation", "constraint",  \
```

Comment:
I don't know if it's always a constraint but I don't have an immediate better target string idea and this is only for tracing, so meh.